src/fns/unconstrained_helpers.nr

```diff
@@ -18,6 +18,13 @@ global TWO_POW_60: u64 = 0x1000000000000000;
  * __tonelli_shanks_sqrt
  */
 
+pub(crate) unconstrained fn __from_field<let N: u32>(field: Field) -> [Field; N] {
+    // cast the field to a u60 representation
+    let res_u60: U60Repr<N, 2> = U60Repr::from_field(field);
+    let result: [Field; N] = U60Repr::into(res_u60);
```

Comment:
briliig execution would be faster if we could go directly to 120 bit limbs rather than going through u60repr.

Reply:
I've put zero thought into how easy this would be to do though.